tables/he-int-g1.utb

```diff
@@ -0,0 +1,446 @@
+# liblouis: Heb Mix grade 1 (EBAE) table
+#
+# -----------
+#
+#+locale:he-en-ar
```

Comment:
Locale "he-en-ar" doesn't make sense. We need to find a proper solution for this. Actually I think first and foremost we need a proper description of what this table is. You say here in the metadata that it is the American braille system. If that would be correct, we should actually merge it with en-us-g1.ctb. However judging from the first line of the table it seems more like it is a mixture of the two. Or maybe is it more something like the "[grc-international-en.utb](https://github.com/liblouis/liblouis/blob/master/tables/grc-international-en.utb)" table, which implements "Greek internationalized braille as used by English speakers".